modules/dxp/apps/search-experiences/search-experiences-web/src/main/java/com/liferay/search/experiences/web/internal/blueprint/admin/portlet/action/GetSitesMVCResourceCommand.java

```diff
@@ -42,18 +40,8 @@ public boolean serveResource(
 		ResourceRequest resourceRequest, ResourceResponse resourceResponse) {
 
 		try {
-			String cmd = ParamUtil.getString(resourceRequest, Constants.CMD);
-
-			JSONObject jsonObject = null;
-
-			if (cmd.equals("getSiteJSONObject")) {
```

Comment:
@felipelorenz the original logic seems to imply that jsonObect should potentially be null (ie if cmd = "helloworld"), so just to keep things the same as they were before, i tihink we can probably add back the single 
`if (cmd.equals("getSiteJSONObject"))` check

Reply:
actually its "getSitesJSONObject" right?

Reply:
Yeah, you're right!